andela/travel_tool_front
# #162080188 Reposition the notification pane

#### What does this PR do?
This PR fixes the position of the notification pane to stop the notification pane from scrolling off the bottom right of the page.

#### Description of Task to be completed?
The PR repositioned the notification pane by setting the notification pane to the bottom right of the page and changing its height to incorporate the nav bar.

#### How should this be manually tested?
- Clone the repo `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the project directory `cd travel_tool_front`
- Check out the **bg-modify-notification-pane-ui-162080188** branch `git checkout bg-modify-notification-pane-ui-162080188`
- Toggle the notification and scroll the page.
- Ensure the notification page always stick to the bottom right.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162080188

#### Screenshots (if appropriate)
**Before**
![screenshot 2018-11-20 12 04 27](https://user-images.githubusercontent.com/10863328/48762633-7d993700-ecbc-11e8-9b72-1d873a4a81fe.png)

**After**
![screenshot 2018-11-20 12 05 16](https://user-images.githubusercontent.com/10863328/48762668-930e6100-ecbc-11e8-9b14-1f9f9c7e5b68.png)

#### Questions:
